ozaria/site/components/teacher-dashboard/BaseMyClasses/ClassComponent.vue

```diff
@@ -24,17 +25,49 @@ export default {
   },
 
   computed: {
+    ...mapGetters({
+      tournamentsByClan: 'clans/tournamentsByClan',
+      tournaments: 'clans/tournaments',
+      allTournamentsLoaded: 'clans/allTournamentsLoaded'
+    }),
+
     showEsportsCampInfoCoCo () {
       return utils.isCodeCombat && me.isCodeNinja() && !this.chapterStats.length
     },
 
+    me () {
+      return me
+    },
+
     showEsportsCampInfoOz () {
       return utils.isOzaria && me.isCodeNinja() && this.chapterStats.length === 2
     },
 
     showJuniorCampInfo () {
       return utils.isCodeCombat && me.isCodeNinja() && this.chapterStats.length === 1
     },
+
+    currentTournaments () {
+      return _.flatten(Object.values(this.tournaments))
+    },
+
+    equinoxTournament () {
+      return this.currentTournaments.find(tournament => tournament.slug === 'equinox') || {}
```

Comment:
This will just get the first one. That's ok for now, but certain centers might have more than one esports camp, so then the later camps would have the tournament link for the first camp, right? Can we do some date matching to find the right one?

Reply:
okay